modules/ding_smartbanner/ding_smartbanner.install

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @file
+ * Install and uninstall ding_smartbanner.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function ding_smartbanner_install() {
+  $smartbanner_config = array(
+    'base_config' => array(
+      'days_hidden' => '15',
+      'days_reminder' => '90',
+      'app_store_language' => 'en',
```

Comment:
Think the installed language should be 'dk' as this is the default DDB CMS language and the app descriptions below is in danish.